doc/release-notes/10169-JSON-schema-validation.md

```diff
@@ -0,0 +1,4 @@
+### JSON Schema for datasets
+
+Enhanced JSON schema validation with checks for required and allowed child objects, Type checking for field types including: ''primative''; ''compound''; and ''controlledVocabulary'' . More user-friendly error messages to help pinpoint the issues in the Dataset JSON.  Rules are driven off the database schema, so no manual configuration is needed. See [Retrieve a Dataset JSON Schema for a Collection](https://guides.dataverse.org/en/6.1/api/native-api.html#retrieve-a-dataset-json-schema-for-a-collection) in the API Guide and PR #10169.
```

Comment:
should the link here reference the 6.2 or 6.3 version of the guide?

Reply:
It should be 6.3. this was a cut and paste flaw. Do we have a link that always posts to the latest? Since 6.3 endpoint doesn't exist yet.

Reply:
modified to 6.3 since I saw that in another md pre release file